src/pages/resultsView/survival/SurvivalChart.tsx

```diff
@@ -183,6 +181,7 @@ export default class SurvivalChart extends React.Component<ISurvivalChartProps,
                 </div>
                 {this.tooltipModel &&
                     <Popover arrowOffsetTop={48} positionLeft={this.tooltipModel.x + 15}
+                             { ...{container:this} }
```

Comment:
basically this causes popover to be placed inside the top level div of this component instead of body.  Since mouseover event seems to give offset relative to parent div, this is necessary.  (Problem was caused by adding header and padding to chart.)
The reason I'm using this funky { ...{ }} syntax is that React-Bootstrap crappy typings don't include this container property.